### PR TITLE
Movie list pages

### PR DESCRIPTION
**Pull request by adim2222**
Working hotloading movielist pages.

### CODE REVIEW

File: src/Routes/MoviesList/components/PageSelector/styled.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    border: none;
+    margin: 6px;
+    padding: 8px;
+    background-color: #D6E4FF;
+    &:hover {
+        background-color: #baccef;
+    }
+`
+
+export const PageCounter = styled.span`
```
Comment (DawidGaski) [inline src/Routes/MoviesList/components/PageSelector/styled.js:19]: here, we need to add : 
`
font-weight: 600;
font-size: 16px;
`  for " 1 " and " 500 "

Comment (DawidGaski) [reply src/Routes/MoviesList/components/PageSelector/styled.js:19]: words " page " and " of "need some ` div` with:
`
color: #7E839A;
` (better with theme)

File: src/Routes/MoviesList/components/PageSelector/PageSelector.js
Comment (DawidGaski) [inline src/Routes/MoviesList/components/PageSelector/PageSelector.js:1]: Nice work, pagination looks great, no corrections needed.
it worth added svg as ReactComponent for better code view

File: src/Routes/MoviesList/index.js [hunk 2]
```diff
@@ -40,18 +44,23 @@ function MoviesList() {
           movie.genre_ids.forEach((id) => {
             movieTags.push(genreTags.find((tag) => id === tag.id).name);
           });
+
+          let rating = 0;
+          rating = movie.vote_average.toFixed(1);
```
Comment (DawidGaski) [inline src/Routes/MoviesList/index.js:49]: nice, you added toFixed (1),
we need it

File: src/Routes/SearchResults/index.js [hunk 3]
```diff
@@ -39,13 +39,17 @@ const SearchResults = ({ query }) => {
                     movie.genre_ids.forEach((id) => {
                         movieTags.push(genreTags.find((tag) => id === tag.id).name);
                     });
+
+                    let rating = 0;
+                    rating = movie.vote_average.toFixed(1);
```
Comment (DawidGaski) [inline src/Routes/SearchResults/index.js:46]: and again, GJ

**Review by DawidGaski — APPROVED** [review]
very well, later we need to add colors by theme, 
check my note, 
Can merge it 
like it.

File: src/Routes/MoviesList/components/PageSelector/styled.js (hunk 1 above)
Comment (adim2222) [reply src/Routes/MoviesList/components/PageSelector/styled.js:19]: ill add theese changes in the merge commit

**Pull request merged**